snapcraft/internal/errors.py

```diff
@@ -50,6 +51,7 @@ def __init__(self, step):
 class StepOutdatedError(SnapcraftError):
 
     fmt = (
+        'Failed to re-use files from previous build: '
         'The {step!r} step of {part!r} is out of date:\n'
         '{report}'
         'In order to continue, please clean that part\'s {step!r} step '
```

Comment:
“In order to” can nearly always be shortened to just “To”.

Reply:
fixed.